kotest-common/src/desktopMain/kotlin/io/kotest/mpp/sysprop.kt

```diff
@@ -1,7 +1,9 @@
 package io.kotest.mpp
 
+import kotlinx.cinterop.ExperimentalForeignApi
 import platform.posix.getenv
 import kotlinx.cinterop.toKString
 
 actual fun sysprop(name: String): String? = null
+@OptIn(ExperimentalForeignApi::class)
```

Comment:
Whats ExperimentalForeignApi do

Reply:
Don't remember exactly, seems to be required when calling this one (which was there before):
```kotlin
/**
 * @return the [kotlin.String] decoded from given zero-terminated UTF-8-encoded C string.
 */
@ExperimentalForeignApi
public fun CPointer<ByteVar>.toKString(): String = this.toKStringFromUtf8()
```

Reply:
All of the foreign language APIs (`CPointer` etc.) [are experimental][1].

[1]: https://kotlinlang.org/api/latest/jvm/stdlib/kotlinx.cinterop/-experimental-foreign-api/